lib/internal/process/esm_loader.js

```diff
@@ -34,14 +34,14 @@ exports.importModuleDynamicallyCallback = async function(wrap, specifier) {
   throw new ERR_VM_DYNAMIC_IMPORT_CALLBACK_MISSING();
 };
 
-let ESMLoader = new Loader();
-exports.ESMLoader = ESMLoader;
+let esmLoader;
+exports.ESMLoader = esmLoader;
 
 async function initializeLoader() {
   const { getOptionValue } = require('internal/options');
-  const userLoader = getOptionValue('--experimental-loader');
-  if (!userLoader)
-    return;
+  const userLoaders = getOptionValue('--experimental-loader');
+  if (!userLoaders.length) return;
+
   let cwd;
   try {
     cwd = process.cwd() + '/';
```

Comment:
Is it possible for `process.cwd()` to fail—if so, how/why? (I'm wondering why the try/catch) I see nothing in the process docs, and a quick google search returns nothin'.

Reply:
I forget exactly which tests these were but because this is a low-level bootstrap, it applies in all contexts - and there was a type of initialization that didn't have process defined here. You can running the tests with it removed, I'm pretty sure that will flag up the ones. And if it doesn't you could remove it :)

Reply:
yes, this can fail if the parent directory is deleted while the process runs.